sql/testdata/builtin_function

```diff
@@ -219,3 +219,89 @@ query T
 SELECT right('abcde', 2)
 ----
 de
+
+# TIMESTAMP/DATE builtins.
+
+query T
+SELECT age(timestamp '2001-04-10 22:06:45', timestamp '1957-06-13')
+----
+384190h6m45s
+
+query T
+SELECT age(timestamp '1957-06-13') - age(now(), timestamp '1957-06-13') < interval '1s'
+----
+true
+
+query T
+SELECT now() - timestamp '2015-06-13' > interval '100h'
+----
+true
+
+query T
+SELECT now() - clock_timestamp() < interval '1s' 
+----
+true
+
+query T
+SELECT now() - current_date < interval '24h1s'
```

Comment:
I think we should pick one time value for the entire query (which would make the time-related functions effectively "pure" for our purposes).

MySQL has a `SET TIMESTAMP` statement that can be used to set the (session scoped) timestamp. I'm not sure this makes sense for us (MySQL needs it because their replication works by replaying statements on followers, so everything needs to be deterministic), but it would make testing time-related functions easier.